### DISCUSSION TITLE
How do you specify font weight (bold) in .json theme file?

### DISCUSSION THREAD

**Discussion by 32-git** [TomSchimansky/CustomTkinter#401] [Q&A] [answered]
As it is stated above, I would like to know how one defines a font to be bold in the json file itself.  Font family and size is clear, just not weight.

I have seen a discussion which was pertaining to the Python file. However, that is not what I where I would like to have it defined, therefore I felt the need to start this discussion. I apologise if I may have missed another similar discussion though.

**Comment by Akascape** [comment, marked as answer]
@32-git It's easy to change the font weight in json file, you have to change this [block of code](https://github.com/TomSchimansky/CustomTkinter/blob/f39ee5764a2979f0c56d15a7da2c1bd4f6cdd838/customtkinter/assets/themes/blue.json#L40) for any font changes:
```
},
  "text": {
    "macOS": {
      "font": "SF Display",
      "size": -13
    },
    "Windows": {
      "font": "Roboto",
      "size": -13
    },
    "Linux": {
      "font": "Roboto",
      "size": -13
    }
  },
```

For bold font weight, just replace the `"font": "Roboto",` to `"font": "Roboto Bold",`

Here is the difference:

![example](https://user-images.githubusercontent.com/89206401/185855352-2f0beb5e-82a4-4432-9313-05cc731729a7.png)

Hope it helps, you can also try my [ThemeMaker](https://github.com/TomSchimansky/CustomTkinter/discussions/264) program for making custom themes.

**Reply by 32-git** [reply]
Hi. Apologies for the late reply, but not every font necessarily has a defined bold version. Therefore, it is best if there is a new feature of sorts, which allows for the changing of font weight regardless of font family.
